soc/arm/atmel_sam0/samr34/Kconfig.defconfig.series

```diff
@@ -0,0 +1,22 @@
+# Atmel SAML MCU series configuration options
+
+# Copyright (c) 2021 Argentum Systems Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_SAMR34
+
+config SOC_SERIES
+	default "samr34"
+
+config SOC_PART_NUMBER
+	default "samr34j16b" if SOC_PART_NUMBER_SAMR34J16B
+	default "samr34j17b" if SOC_PART_NUMBER_SAMR34J17B
+	default "samr34j18b" if SOC_PART_NUMBER_SAMR34J18B
+
+config NUM_IRQS
+	default 23
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 48000000
```

Comment:
Try pick everything you can from devicetree as this https://github.com/zephyrproject-rtos/zephyr/blob/44462723f59ddcc0786af0b5406122476c2addf9/soc/riscv/riscv-privilege/gd32vf103/Kconfig.defconfig.gd32vf103#L12-L13